### PR TITLE
Update README and appdata files

### PR DESCRIPTION
**Pull request by ovari**

### CODE REVIEW

**Comment by ovari** [comment]
Added some links to the appdata file.

File: README.md [hunk 1]
```diff
@@ -8,14 +8,28 @@ Betterbird is a fine-tuned version of [Mozilla Thunderbird](https://www.thunderb
...
 
-#### Migration from pre-exisiting non-flatpak installations
-In order to migrate from pre-exisiting non-flatpak installation and preserve all settings please copy or move entire<br>
+#### Migration from pre-exisiting installations
+
+##### Migration from pre-exisiting Thunderbird flatpak installations
+In order to migrate from pre-exisiting Thunderbird flatpak installation and preserve all settings please copy or move entire<br>
+`~/.var/app/org.mozilla.Thunderbird/.thunderbird`<br>
+folder into<br>
+`~/.var/app/eu.betterbird.Betterbird/.betterbird`
```
Comment (mfschumann) [inline README.md]: This should be `~/.var/app/eu.betterbird.Betterbird/.thunderbird`. Betterbird does not find the profile if it is in `.betterbird` folder.

File: README.md [hunk 2]
```diff
@@ -8,14 +8,28 @@ Betterbird is a fine-tuned version of [Mozilla Thunderbird](https://www.thunderb
...
+##### Migration from pre-exisiting Thunderbird non-flatpak installations
+In order to migrate from pre-exisiting non-flatpak Thunderbird installation and preserve all settings please copy or move entire<br>
 `~/.thunderbird`<br>
 folder into<br>
-`~/.var/app/org.mozilla.Thunderbird/.thunderbird`
+`~/.var/app/eu.betterbird.Betterbird/.betterbird`
+
+##### Migration from pre-exisiting Betterbird non-flatpak installations
+In order to migrate from pre-exisiting non-flatpak Betterbird installation and preserve all settings please copy or move entire<br>
+`~/.betterbird`<br>
```
Comment (mfschumann) [inline README.md]: Should be `~/.thunderbird` I think.

**Review by mfschumann — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (ghost) [reply README.md]: Done

File: README.md (hunk 1 above)
Comment (ghost) [reply README.md]: Done

**Review by mfschumann — APPROVED** [review]
Thanks, looks good!

**Pull request merged**